### PR TITLE
Add workflow to test markdown and later push changes to the wiki on t…

### PR DESCRIPTION
**Pull request by miigotu**
…his repo

Signed-off-by: miigotu <miigotu@gmail.com>

### CODE REVIEW

File: How-to-use-Download-url.md [hunk 1]
```diff
@@ -11,35 +12,37 @@ You do need a separate webserver with a vhost configured to make the shows avail
...
        Order deny,allow
        Allow from all
     </Directory>
-  </VirtualHost>
 
+  </VirtualHost>
 
 <span class="section">'''nginx'''</span>
 Example configuration for nginx :
-  server
```
Comment (miigotu) [inline How-to-use-Download-url.md:28]: @BKSteve this should be in a code block

**Comment by miigotu** [comment]
@BKSteve im not sure this is a good linter, but I'd like to do something automated like this. People then can edit a file with a PR here, and it will get linted, then when merged automatically pushed to the actual wiki. Later, we can have it automatically push to the actual sickchill wiki.

This lets non-members contribute to the wiki without us making it publicly editable, while still not having to give write permissions to sickchill for every person that wants to make changes.

**Comment by BKSteve** [comment]
There's certainly got to be better ways to do markdown. Did these with IJ.

**Comment by miigotu** [comment]
> There's certainly got to be better ways to do markdown. Did these with IJ.

It works great with intellij, just need to add the right plugins and ad a .editor config to this project. It was just a wiki so it didn't have any testing or config for anything so it made it difficult. Awesome that the wiki repos can hold other file types though haha, I was worried it wouldn't work

**Comment by miigotu** [comment]
Locally you can format files before pushing. The build failing means your formatting is not right. 
```
npm install -g prettier
npx prettier -W . 
```

I'll see about adding an .editorconfig to this repo, and maybe a precomit-hooks configuration so we can always make sure it's right. I wonder if black works on markdown, I'd rather that since it is more configurable.

**Comment by miigotu** [comment]
We should use some of these:
https://earthly.dev/blog/markdown-lint/

How I'm setting this up is a totally new thing, I've never seen anyone do this with their wiki to make pull requests that end up right on the wiki possible. It's kind of cool. Now I'm starting to get into the idea of using tools to make our docs/wiki really good, and I want to start generating docs for the code and settings so we can put it on readthedocs.

File: How-to-use-Download-url.md (hunk 1 above)
Comment (BKSteve) [reply How-to-use-Download-url.md:28]: Just a move from old one.

Comment (miigotu) [reply How-to-use-Download-url.md:28]: Yeah just noticing things we can fix, I didn't think you messed it up lol

**Pull request merged**